### PR TITLE
Move theme support from Beta to Main

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #64, #70

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Move theme support from beta to main, enhancing the component with theme capabilities. Improve performance and update development scripts, including pre-commit hooks. Clean up documentation by removing obsolete sections.

New Features:
- Introduce support for component themes, moving the feature from beta to the main branch.

Enhancements:
- Improve performance by adding a messaging script and updating the commit message path.
- Update pre-commit hook and husky prepare command for better development workflow.

Documentation:
- Remove outdated documentation regarding CSS custom property fallbacks from the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,15 +58,6 @@ The use of any Auro custom element has a dependency on the [Auro Design Tokens](
...
 <!-- AURO-GENERATED-CONTENT:END -->
 
-### CSS Custom Property fallbacks
-
-<!-- AURO-GENERATED-CONTENT:START (REMOTE:url=https://raw.githubusercontent.com/AlaskaAirlines/WC-Generator/master/componentDocs/partials/usage/cssFallbacks.md) -->
-[CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) are [not supported](https://auro.alaskaair.com/support/custom-properties) in older browsers. For this, fallback properties are pre-generated and included with the npm.
-
-Any update to the Auro Design Tokens will be immediately reflected with browsers that support CSS custom properties, legacy browsers will require updated components with pre-generated fallback properties.
-
-<!-- AURO-GENERATED-CONTENT:END -->
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply README.md:68]: Not addressing at this time.

File: src/tokens.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+
+:host {
+  --ds-auro-card-border-color: var(--ds-color-border-divider-default, #{$ds-color-border-divider-default});
+  --ds-auro-card-link-border-color: transparent;
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply src/tokens.scss:5]: Not changing name at this time.

File: .github/settings.yml [hunk 3]
```diff
@@ -111,83 +107,64 @@ labels:
...
   - name: 'Abandoned'
-    color: '156fad'
     description: The author of this issue or Pull Request is not responding
     aliases:
     - wontfix
     - invalid
-  - name: 'Type: Bug'
-    color: df0b37
-    description: Bug or Bug fixes
+  - name: 'Type: Bug'   description: Bug or Bug fixes
```
Comment (jason-capsule42) [inline .github/settings.yml:129]: Line break deleted by accident

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**